src/content/1.7/development/webservice/tutorials/3-prestashop-webservice-lib/1-install-library.md

```diff
@@ -0,0 +1,116 @@
+---
+title: 1 - Accessing the Webservice
```

Comment:
```suggestion
title: 1 - PrestaShop Webservice lib
```

Note that the title and the file name should be the same

Reply:
You're right, but if we use `PrestaShop Webservice lib` it will be redundant with the parent section `3 - PrestaShop Webservice lib > 1 - PrestaShop Webservice lib`

The problem is this page actually deals with two topics: Install the library + Access the webservice

Reply:
It's a suggestion, you can choose the one you think it's best. Just try to keep  the url and the page title consistent.

Reply:
I changed it to `Setup library` which can englobe both topics